### ISSUE TITLE
Change hard-coded query language option in the semantic reranker into a variable

### ISSUE THREAD

**Issue by marcopostmovares** [microsoft/PubSec-Info-Assistant#467]
In the file /app/backend/approaches/chatreadretrieveread.py, there is some Python code to configure hybrid semantic search using the semantic reranker. The configuration of the query language on line 228 in that file is currently hard-coded to "en-us". Could you please make use of configurable variables instead of hard-coded strings?

```python
query_language="en-us",
 # query_language=self.query_term_language,
```

**Comment by dayland** [comment]
@marcopostmovares, we are working on more explicit language support in our next release. I have marked this a bug and assigned to @ArpitaisAn0maly who is working on this feature.

**Comment by ArpitaisAn0maly** [comment]
Hello @marcopostmovares,

A PR for this merged into VNext-Dev. 
https://github.com/microsoft/PubSec-Info-Assistant/pull/470

Please note that query_language is no longer needed for Semantic Reranking as it is language agnostic now.

**Comment by dayland** [comment]
Resolved with latest release [v1.1](https://github.com/microsoft/PubSec-Info-Assistant/releases/tag/v1.1)
